src/test/java/performance/users.feature

```diff
@@ -0,0 +1,46 @@
+Feature: sample karate test script
```

Comment:
Please remove this file and use the actual users file in performance

Reply:
Removed and referenced the actual file